policy/lib/refs.rego

```diff
@@ -36,6 +36,8 @@ task_ref(task) := i {
 		"url": _param(r, "url", ""),
 		"revision": _param(r, "revision", ""),
 		"pathInRepo": _param(r, "pathInRepo", ""),
+		"name": _param(r, "pathInRepo", ""),
```

Comment:
I'm looking at a attestation with the git resolver, and the task has a `name` property, could we use that instead?

```suggestion
		"name": object.get(r, "name", ""),
```

Reply:
Yes, good catch.

Reply:
Actually... `r` here should be the [taskRef](https://gist.github.com/zregvart/7cc23e92ba37333f70c07af7671cac5a#file-attestation-v1-json-L21). This is always going to be an empty string.

Reply:
This is actually the taskRef from the taskRun that's stored in resolvedDependencies. The information from externalParameters can change based on the pipeline type. inline, etc